src/engine/LastPlayedVersion.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using Godot;
+
+/// <summary>
+///   Loads and allows updating the last played Thrive version
+/// </summary>
+public static class LastPlayedVersion
+{
+    private static Lazy<string?> lastPlayed = new(ReadLastPlayedVersion);
+
+    private static bool handledCurrentVersionPlayed;
+
+    public static string? LastPlayed => lastPlayed.Value;
+
+    public static void MarkCurrentVersionAsPlayed()
```

Comment:
Microbe freebuild editor doesn't call this

Reply:
I'll move the call to mark the version as played to `MainMenu.OnEnteringGame` to not have to put it all over (hopefully entering freebuild from auto-evo exploring tool also calls that).

Reply:
Nope. Auto-evo exploring tool is a separate scene and doesn't hold any info about the main menu.

(Oh, there's also thrivopedia)

Reply:
I opened an issue as it's pretty important to have all methods of starting a game trigger the same operations for consistency: https://github.com/Revolutionary-Games/Thrive/issues/4112